<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WallWnd</class>
 <widget class="QWidget" name="WallWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Wall</string>
  </property>
  <widget class="QComboBox" name="cbChn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbRing">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>500</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cbWnd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>500</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="wallView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1021</width>
     <height>481</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="dragMode">
    <enum>QGraphicsView::RubberBandDrag</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="x">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="y">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="w">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="h">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ax">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ay">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="aw">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ah">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>500</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>w:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>500</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>h:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbActivateRing">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>信号走通到环-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCloseWnd">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbResetWnd">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>540</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>调整窗口</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbDragWnd">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>控制窗口</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
